.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build and test a pull request
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ci:
+    name: Run CI build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.14
+
+      - name: Install github.com/vbatts/git-validation
+        working-directory: /tmp
+        run: go get -u github.com/vbatts/git-validation
+
+      - name: Install github.com/cpuguy83/go-md2man
+        env:
+          VERSION: "v1.0.10"
+        working-directory: /tmp
+        run: go get "github.com/cpuguy83/go-md2man@${VERSION}"
+
+      - name: Check out source code
+        uses: actions/checkout@v2
+
+      - name: Validate DCO
+        env:
+          DCO_VERBOSITY: "-q"
+          GIT_CHECK_EXCLUDE: "./vendor:./script/validate/template"
+        run: git-validation "${DCO_VERBOSITY}" -run DCO,short-subject,dangling-whitespace
```

Comment:
DCO can be handled directly through the [DCO Probot App](https://probot.github.io/apps/dco/) like we currently do on moby [or buildx](https://github.com/docker/buildx/pull/489/checks?check_run_id=1580980809):

![image](https://user-images.githubusercontent.com/1951866/102686237-343ead00-41de-11eb-9541-69dacf1f23d2.png)

Just need to enable it for this repo.

Reply:
Sounds good, as a first step I wanted to "copy" the existing travis build process and iterate on it once this repo has something working. The sooner this repo gets off travis the better.